CMakeLists.txt

```diff
@@ -544,10 +544,7 @@ IF (CXXTEST_FOUND)
 		)
 	ENDIF (CMAKE_BUILD_TYPE STREQUAL "Coverage")
 	ADD_CUSTOM_TARGET(test_python
-		DEPENDS PythonEvalUTest
-				PythonUtilitiesUTest
-				bindlink_cython atomspace_cython
-				scheme_wrapper logger_cython utilities_cython
+		DEPENDS tests
```

Comment:
Why test_python depends on tests? 
Should it be just removed instead?

Reply:
@inflector Do you think we can remove test_python and run python tests together with all other tests?